helpers/base.py

```diff
@@ -110,3 +112,35 @@ def fetch_values_by_datatype(rec, datatype):
         results.append(result)
 
     return results
+
+def is_ioc(rec):
+    """Detect is any data in a record matching to known IOC"""
+    intel = StreamThreatIntel.get_intelligence()
+    datatypes_ioc_mapping = StreamThreatIntel.get_config()
+
+    if not (datatypes_ioc_mapping and rec.get('normalized_types')):
+        return False
+
+    for datatype in rec['normalized_types']:
+        if datatype not in datatypes_ioc_mapping:
+            continue
+        results = fetch_values_by_datatype(rec, datatype)
+        for result in results:
+            if (intel.get(datatypes_ioc_mapping[datatype])
+                    and result in intel[datatypes_ioc_mapping[datatype]]):
+                if 'streamalert:ioc' in rec:
```

Comment:
side note: `streamalert:ioc` should probably be a const

Reply:
Addressed with: https://github.com/airbnb/streamalert/pull/327/commits/4296f6400cc43809cbf7b4bfbde4b678d0dc1a09